tests/test_quantum_info_clifford.py

```diff
@@ -17,32 +17,32 @@
 
 def construct_clifford_backend(backend):
     if isinstance(backend, TensorflowBackend):
-        with pytest.raises(NotImplementedError) as excinfo:
-            clifford_backend = CliffordBackend(backend)
-            assert (
-                str(excinfo.value)
-                == "TensorflowBackend for Clifford Simulation is not supported yet."
-            )
+        with pytest.raises(NotImplementedError):
+            clifford_backend = CliffordBackend(backend.name)
     else:
+        backend = backend.name if backend.platform is None else backend.platform
```

Comment:
This is another instance of `get_engine()` (with a compensating negation and swap)